contrib/ckptfile/README

```diff
@@ -1,2 +1,47 @@
-This plugin serves as a template.  Please modify the given
-functions according to your needs.
+# 'ckptfile' plugin
+
+This plugin allows you to:
+
+ a) specify a file that DMTCP should checkpoint; and
+ b) specify the path on restart of a file if it wasn't checkpointed.
+
+The plugin reads a user-specified database at checkpoint time. The
+database is a simple text file that specifies the files that should
+be checkpointed, and the restart paths of files that shouldn't be
+checkpointed.
+
+The database file's format is defined below.
+
+   file:ckpt={1|0};rstrt=file
+   file-pattern:ckpt={1|0};rstrt=file-pattern
```

Comment:
I have several comments and questions here:

1. Shouldn't we be writing FILE and FILE_PATTERN to emphasize that these are meta-names that should be replaced with an actual filename?
2. Would it be easy to modify the code to use true and false, and change the README to:  FILE:ckpt={true|false};... (and similarly for FILE_PATTERN?
3. You never say what is a FILE_PATTERN.  Is it a regular expression?
4. In `file:ckpt={1|0};rstrt=file`, do the two occurences of the keyword, `file`, refer to the same filename?  If so, why repeat `file`?  If not, then why do you later write `the saved copy of the file is restored at its original`?
5. How does the user know if something like `a.txt`will be treated as a raw name, or as a regular expression with `.` matching any single character?  (This assumes that `file-pattern` means a regular expression.)

Have you considered an alternative syntax, such as:
```
SAVE_AND_RESTORE: {FILE|FILE_PATTERN}
FILE_WILL_BE_UPDATED_BEFORE_RESTART: {FILE|FILE_PATTERN}
```